composer.json

```diff
@@ -4,14 +4,28 @@
   "type": "mautic-plugin",
   "version": "1.0.1",
   "keywords": ["mautic","plugin","integration"],
+  "autoload": {
+    "psr-4": {"MauticPlugin\\HelloWorldBundle\\": ""}
+  },
   "require": {
-    "mautic/composer-plugin": "^1.0",
-    "php": "^7.2"
+    "php": ">=7.4, <8.1",
+    "guzzlehttp/psr7": "^2.0",
+    "mautic/core-lib": "^4.0"
+  },
+  "extra": {
+    "install-directory-name": "HelloWorldBundle"
   },
   "require-dev": {
-    "phpstan/phpstan": "^0.11.12",
-    "symplify/easy-coding-standard": "^6.0"
+    "phpstan/phpstan": "^0.12",
+    "symplify/easy-coding-standard": "^6.0",
+    "phpunit/phpunit": "^9.5"
```

Comment:
Yes, because I'm trying to allow github to run tests on this bundle (as suggested by https://mautic-developer.readthedocs.io/en/latest/marketplace/best_practices.html 5th point).

Reply:
I think that tests run via CI are essential. It's great that you are working on this!

What I meant to say is that you cannot run the tests on this repository alone, right? Because the Mautic namespaces wouldn't exist. You firstly need Mautic to install, then install this plugin into it and then you can run the tests. When Mautic is present then it has PHPUNIT, PHPSTAN and CS Fixer in it already. It is already configured. So there is no need to do it in each plugin separately.